.dockerignore

```diff
@@ -0,0 +1,27 @@
+*
+# IoT Event Analytics: Node.js SDK
+!iot-event-analytics/package.json
```

Comment:
Unfortunately, we have to exclude all at once and the re-include the ones, we actually need. Masking it this way already saves space instead of explicitly excluding paths